import_tracker/web_client/stylesheets/importList.styl

```diff
@@ -23,6 +24,9 @@
 
     td
       padding 3px 0
+      white-space: nowrap;
```

Comment:
This makes it necessary to hover over the lien to see the actual value.  Many values will start with the same /collection/foo/... so this reduces the utility of the view.  I think it would be better to remove the `white-space` directive, and the add some horizontal padding.  Further, I'd revert the `table-layout: fixed` and just add some padding to the `th` elements to ensure some space between columns.

Reply:
changed in 85d05d1a60a4ab6f0527729099e638699a233957